staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -51,6 +57,7 @@ func init() {
 // To add a new feature, define a key for it above and add it here. The features will be
 // available throughout Kubernetes binaries.
 var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureSpec{
-	StreamingProxyRedirects: {Default: true, PreRelease: utilfeature.Beta},
-	AdvancedAuditing:        {Default: false, PreRelease: utilfeature.Alpha},
+	StreamingProxyRedirects:      {Default: true, PreRelease: utilfeature.Beta},
+	AdvancedAuditing:             {Default: false, PreRelease: utilfeature.Alpha},
+	EnableAPIResponseCompression: {Default: false, PreRelease: utilfeature.GA},
```

Comment:
I'm a little unsure about a feature that isn't particularly alpha, but also isn't on by default... do we have a plan for permanent gates like this?